content/en/docs/demos/certificate_rotation.md

```diff
@@ -0,0 +1,181 @@
+---
+title: "Rotating the MeshRootCertificate"
+description: "Using the OSM CLI to rotate the MeshRootCertificate"
+type: docs
+weight: 22
+---
+
+This guide demonstrates how to use the [MeshRootCertificate](../guides/certificate_management/certificate-rotation.md) preview feature with the OSM CLI. 
+
+>  WARNING:  This feature is currently in preview and requires the `EnableMeshRootCertificate` feature flag enabled in the MeshConfig.  
+
+## Prerequisites
+
+- Kubernetes cluster running Kubernetes {{< param min_k8s_version >}} or greater.
+- Have `kubectl` available to interact with the API server.
+- Have `osm` CLI available for installing and managing the service mesh.
+- Have `openssl` available to view certificate information.
+
+## Demo
+
+The following demo shows how to use the OSM CLI to initiate a root certificate rotation using the Tresor certificate provider.  Learn more about this process in the [Certificate Rotation documentation](../guides/certificate_management/certificate-rotation.md).
+
+1. Install OSM with the `EnableMeshRootCertificate` [feature flag](../api_reference/config/v1alpha2.md#featureflags) set to true.
+   ```console
+   osm install --set="osm.featureFlags.enableMeshRootCertificate=true"
+   ```
+
+1. Check that the default `MeshRootCertificate` was created and the role is `active`:
+   ```console
+   kubectl get mrc -n osm-system  # Replace osm-system with the namespace where OSM is installed
```

Comment:
```suggestion
   $ kubectl get mrc -n osm-system  # Replace osm-system with the namespace where OSM is installed
```

Reply:
looks like this is an inconsistency in the OSM docs in general.  I can clean it up for this doc (I prefer no $) but see something like https://release-v1-2.docs.openservicemesh.io/docs/demos/local_rate_limit_http/

Reply:
removed all the `$'s from this doc